src/nbl/ui/CWindowX11.cpp

```diff
@@ -36,92 +36,90 @@ CWindowManagerX11::CWindowManagerX11()
 void CWindowX11::processEvent(XEvent event)
 {
     IEventCallback* eventCallback = getEventCallback();
-/*
+
+    std::cout << event.type << std::endl;
+
     switch(event.type)
     {
-        case ConfigureNotify:
+        case ResizeRequest: // window resize event
         {
-            XConfigureEvent e = event.xconfigure;
-            // Resized
-            if(e.width != m_width || e.height != m_height)
+            std::cout << "Resize event!\n";
+            XResizeRequestEvent& e = (XResizeRequestEvent&)event;
+            if (e.width != m_width || e.height != m_height)
             {
-                if(eventCallback->onWindowResized(this, e.width, e.height))
+                if (eventCallback->onWindowResized(this, e.width, e.height))
+                {
                     x11.pXResizeWindow(m_dpy, m_native, e.width, e.height);
-                    std::cout << "Window resized: " << e.width << " " << e.height << std::endl;
-
+                }
             }
-            // Moved
-            if(e.x != m_x || e.y != m_y)
+            break;
+        }
+
+        case ConfigureNotify:  // window move event
+        {
+            std::cout << "Move event!\n";
+            XConfigureRequestEvent& e = (XConfigureRequestEvent&)event;
+             if (e.x != m_x || e.y != m_y)
             {
-                if(eventCallback->onWindowMoved(this, e.x, e.y))
+                if (eventCallback->onWindowMoved(this, e.x, e.y))
+                {
                     x11.pXMoveWindow(m_dpy, m_native, e.x, e.y);
+                }
+            }
+            break;
+        }
+
+        case MapNotify: // window show event
+        {
+            std::cout << "Show event!\n";
```

Comment:
the example 52 (or some similar number) already has all the logging for these window events, just use the callback right

Reply:
Yea, i forgot to delete it. It was just for testing.